HackIllinois/Controllers/ViewControllers/HIUserDetailViewController.swift

```diff
@@ -106,8 +107,19 @@ extension HIUserDetailViewController {
         .onCompletion { result in
             switch result {
             case .success(let data):
-                guard let pass = try? PKPass(data: data),
-                    let vc = PKAddPassesViewController(pass: pass) else { return }
+                let vc: PKAddPassesViewController!
```

Comment:
This shouldn't be optional (remove the `!`). You will need to add a check that PKAddPassesViewController(pass:) returns .some. (ie not nil)